packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -63,45 +63,79 @@ type ExpandableBlockProps = {
   disabled?: boolean;
 };
 
-/**
- * Container that allows content to be hidden behind a brief title and a caption.
- * @example
- * <ExpandableBlock title='Basic Block'>Content in block!</ExpandableBlock>
- * <ExpandableBlock title='Basic Block' caption='basic caption'>Content in block!</ExpandableBlock>
- * <ExpandableBlock title='Default Expanded Block' caption='basic caption' isExpanded={true}>Content in block!</ExpandableBlock>
- * <ExpandableBlock title='Positive status' status='positive'>Content</ExpandableBlock>
- * <ExpandableBlock title='Block with icon' endIcon={<SvgPlaceholder />}>Content</ExpandableBlock>
- */
-export const ExpandableBlock = React.forwardRef((props, ref) => {
+const ExpandableBlockComponent = React.forwardRef((props, forwardedRef) => {
   const {
-    caption,
     children,
     className,
-    title,
     onToggle,
     style,
-    isExpanded = false,
-    endIcon,
+    isExpanded,
     status,
     size = 'default',
     styleType = 'default',
     disabled = false,
     ...rest
   } = props;
 
-  const icon = endIcon ?? (status && StatusIconMap[status]());
-
   const [expanded, setExpanded] = React.useState(isExpanded);
   React.useEffect(() => {
     setExpanded(isExpanded);
   }, [isExpanded]);
```

Comment:
Unresolving this comment because I think you've reverted part of this change.

The logic here is that `isExpanded` (prop) should always be preferred over the internal state. If the user wants default uncontrolled behavior, then `isExpanded` prop should not be passed.

Reply:
hmm not sure I understand correctly. do you mean that the user controls the states themselves via `isExpanded` prop and their own `onClick`? 
the reason I reverted this was because the expanding and collapsing logic relies on the variable controlled by `useState` (which is `expandedState` in the suggested update). Leaving them as `expanded` won't change inner logic when clicking on header

Reply:
> the user controls the states themselves via `isExpanded` prop and their own `onClick`?

Yes but only if they want to opt into controlled mode. By default it will work correctly as uncontrolled.

> Leaving them as expanded **won't** change inner logic when clicking on header

It **will** change if `isExpanded` prop is not passed.


You can test this snippet in vite playground (note there is no `isExpanded` prop passed):

```jsx
<ExpandableBlock>
  <ExpandableBlock.Header label='Expandable block' />
  <ExpandableBlock.Content>Content in block!</ExpandableBlock.Content>
</ExpandableBlock>
```

---

I think in storybook you are explicitly setting value of `isExpanded` in args. That needs to be removed if you want correct uncontrolled behavior.

Reply:
Ok yea those controls made me think the logic was off. I reverted this change back. Also, removed `isExpanded` toggle logic for Accordion Story since that seems to not be needed anymore?